app/models/mobile_user.rb

```diff
@@ -1,6 +1,9 @@
 class MobileUser < ApplicationRecord
   has_many :mobile_comments, foreign_key: :id
   belongs_to :auth_token
+  validates :username, :auth_token, presence: true
+
+  alias_attribute :username, :name
```

Comment:
`username` i `name` to raczej nie to samo. Jeżeli bardzo chcesz alias, który bardziej pasuje w kontekście komentarza, to może `nickname`?